common/src/main/java/org/broadleafcommerce/common/audit/AbstractAuditableListener.java

```diff
@@ -85,17 +85,20 @@ protected void setAuditUpdateData(Object entity, Object auditableObject) throws
     protected void setAuditData(Object entity, Object auditableObject, String dateField, String userField) throws Exception {
         if (entity.getClass().isAnnotationPresent(Entity.class)) {
             Field field = BLCFieldUtils.getSingleField(entity.getClass(), getAuditableFieldName());
-            field.setAccessible(true);
-            if (field.isAnnotationPresent(Embedded.class)) {
-                Object auditable = field.get(entity);
-                if (auditable == null) {
-                    field.set(entity, auditableObject);
-                    auditable = field.get(entity);
-                }
+            if (field != null) {
+                field.setAccessible(true);
+                if (field.isAnnotationPresent(Embedded.class)) {
+                    Object auditable = field.get(entity);
+                    if (auditable == null) {
+                        field.set(entity, auditableObject);
+                        auditable = field.get(entity);
+                    }
+
                 Field temporalField = auditable.getClass().getDeclaredField(dateField);
```

Comment:
@riteshadhikari17 can you indent these? it looks like they are misaligned with the if statement that it is in.